lib/repository/local.rb

```diff
@@ -41,6 +47,12 @@ def retrieve_all_fields
         db.fetch sql
     end
 
+    def retrieve_id_and_date
+        sql = "SELECT id, created_at from #{table} ORDER BY created_at DESC LIMIT 6"
```

Comment:
why 6? What does 6 represent?  Is it to get the last 3 turns of each player?

Reply:
I'm using 6 arbitrarily to limit the amount of suspended games I retrieve

Reply:
It may make sense to extract this as a constant to describe this